src/convenience/frameworks.ts

```diff
@@ -85,13 +85,23 @@ export type AzureAdapter = (context: {
         [key: string]: any;
     };
 }, request: { body?: unknown }) => ReqResHandler;
-export type AzureAdapterV4 = (request: Request, context: {
-    res?: {
-        status: number;
-        body: string;
-        headers?: Record<string, string>;
-    };
-}) => ReqResHandler<{ status: number; body?: string } | { jsonBody: string }>;
+export type AzureV4HttpRequest = {
```

Comment:
We also don't need to export this, it isn't used elsewhere :)
```suggestion
type AzureV4HttpRequest = {
```

Reply:
Ok, thanks